### DISCUSSION TITLE
Prettier not formatting code that uses the satisfies operator in TypeScript

### DISCUSSION THREAD

**Discussion by Sajarin-M** [prettier/prettier#14139] [Q&A]
I am using Prettier 2.8.1 and I have recently encountered an issue where Prettier is not properly formatting my code when it includes the satisfies operator in TypeScript. I have noticed that when I comment out the lines of code that use the satisfies operator, Prettier works as expected. I apologize if this is a trivial issue, but I have looked through the Prettier documentation and I was unable to find any information on how to configure Prettier to support the satisfies operator. 

I was wondering if there is any way to configure Prettier to support the satisfies operator in my code, or if there are any known workarounds for this issue. I would greatly appreciate any guidance you can provide. 

Thank you in advance for your time and help.

**Comment by seiyab** [comment]
It have been resolved as of current stable release. https://github.com/prettier/prettier/pull/13764
I'm wondering why it doesn't look  written in CHANGELOG explicitly.

**Reply by fisker** [reply]
For non-patch release, blog post is used instead. https://prettier.io/blog/2022/11/23/2.8.0.html#typescript
